book/templating.rst

```diff
@@ -1027,46 +1027,12 @@ assets won't be cached when deployed. For example, ``/images/logo.png`` might
 look like ``/images/logo.png?v2``. For more information, see the :ref:`ref-framework-assets-version`
 configuration option.
 
-.. _`book-templating-version-by-asset`:
```

Comment:
Does it make sense to add a warning that this is not supported? Otherwise, people that who use old links might get lost as the anchor wouldn't match anymore.